scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -263,22 +263,14 @@ export {
 	                                          label_values: labels_vector,
 	                                          measurement: double): bool;
 
-	## Telemetry sync hook.
+	## Interval at which the :zeek:see:`Telemetry::sync` hook is invoked.
 	##
-	## This hook is invoked every :zeek:see:`Telemetry::sync_interval`
-	## for script writers to synchronize or mirror metrics with the
-	## telemetry subsystem. For example, when tracking table or value
-	## footprints with gauges, the value in question can be set on an actual
-	## :zeek:see:`Telemetry::Gauge` instance during execution of this hook.
+	## By default, the hook is invoked on demand, setting this option to
+	## a positive interval allows to invoke it regularly, too.
```

Comment:
```suggestion
	## a positive interval allows to invoke it regularly, too. Note that regular
	## invocations are relative to Zeek's network time.
```

Reply:
I'd think this applies to most every interval we have anywhere and may or may not be problematic. But yeah, can add it, given the external visibility.